pulp_rpm/app/models/distribution.py

```diff
@@ -90,6 +92,8 @@ class DistributionTree(Content):
     discnum = models.IntegerField(null=True)
     totaldiscs = models.IntegerField(null=True)
 
+    digest = models.CharField(max_length=64, null=False)
```

Comment:
I have not added it to the serializer. Do we want it to be exposed to users? Should I add it?

Reply:
My inclination is no, not unless we have a compelling reason to do it.